include/bluetooth/mesh/sensor.rst

```diff
@@ -322,10 +331,13 @@ Example: Average ambient temperature in a period of day as a sensor series:
 Sensor settings
 ***************
 
-The list of settings a sensor supports should be set on init. The list should be constant throughout the sensor's lifetime, and may be declared ``const``.
+The list of settings a sensor supports should be set on init.
+The list should be constant throughout the sensor's lifetime, and may be declared ``const``.
 Each entry in the list has a type and two access callbacks, and the list should only contain unique entry types.
 
-The :cpp:member:`bt_mesh_sensor_setting::get` callback is mandatory, while the :cpp:member:`bt_mesh_sensor_setting::set` is optional, allowing for read-only entries. The value of the settings may change at runtime, even outside the ``set`` callback. New values may be rejected by returning a negative error code from the ``set`` callback.
+The :cpp:member:`bt_mesh_sensor_setting::get` callback is mandatory, while the :cpp:member:`bt_mesh_sensor_setting::set` is optional, allowing for read-only entries.
+The value of the settings may change at runtime, even outside the ``set`` callback.
+New values may be rejected by returning a negative error code from the ``set`` callback.
```

Comment:
I'm getting the following errors when building locally (after rebuilding all the docsets):
![image](https://user-images.githubusercontent.com/41291385/77549075-ae404080-6eaf-11ea-8d53-02bf0045bf20.png)

In the output, I see the following:
![image](https://user-images.githubusercontent.com/41291385/77549703-71c11480-6eb0-11ea-8fb1-99c120b8f0b4.png)
(this looks similar for all 4 errors reported)

Reply:
This appears to be [this issue](https://github.com/sphinx-doc/sphinx/issues/2683), fixed in sphinx 1.8. @ru-fu what version of sphinx does ncs use for building public documentation? Any chance of bumping the version number? Also, any suggestions for why it's not broken in [current public documentation](http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/bluetooth/mesh/sensor.html#_CPPv4N24bt_mesh_sensor_threshold5deltaE)?

Reply:
The Sphinx version is defined in zephyr/scripts/requirements.txt:
sphinx>=1.7.5, != 2.4.0

@carlescufi should it be updated?

Reply:
Bump

This PR does not introduce any regressions, the issue Greg is seeing is reproducible in current master as well with Sphinx < 1.8.

Reply:
Yes, the upstream requirements need to be fixed. I will send a PR there.

Reply:
Actually, we still see those upstream even with the latest Sphinx/breathe. This has just not been solved properly in sphinx. We need to use the whitelisting (known issues) warning feature that upstream has to fix this.

Reply:
In fact, this was never fixed properly in Sphinx/breathe as I mention. There are still multiple warnings that we filter out in [zephyr](https://github.com/zephyrproject-rtos/zephyr/tree/master/.known-issues/doc), and we'll probably need to do the same upstream

Reply:
@carlescufi This doesn't break in the CI, just in local builds with a specific version of Sphinx. So agreeing on a specific Sphinx version to use would fix it - without the overhead of adding the filtering script to the nrf build.

Reply:
OK, I see. But sooner or later we will face the issues that upstream does with breathe. For now, locking the version to whatever CI has should do the trick.

Reply:
@ru-fu feel free to send a PR that documents the version of sphinx, breathe and docutils to be used, even if `requirements.txt` uses a floating one to avoid forcing this on everybody.

Reply:
@carlescufi The only combination that we can be sure works is the one used by CI. Putting any combination in the docs that isn't regularly being tested will not help.
So we're back at the (currently stalled) discussion on how to get the tested tools versions in the docs. 

And while putting a set of "known to work" versions in the docs is a start, I think it's pretty bad usability to allow for version combinations that don't work in requirements.txt.

Reply:
Yes, I agree with you, but we inherit the sphinx and breathe versions from zephyr don't we? So I am not sure how we can fix that short of redefining the version ourselves in our own `requirements.txt`